Packs/PrismaAccess/Playbooks/playbook-Prisma_SASE_-_Add_IPs_to_Static_Address_Group.yml

```diff
@@ -0,0 +1,587 @@
+      playbookName: Prisma SASE - Create Address Object
+      type: playbook
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "6"
+    scriptarguments:
+      AutoCommit:
+        simple: Else
+      Folder:
+        complex:
+          root: inputs.Folder
+      IP:
+        complex:
+          root: inputs.IP
+          filters:
+          - - operator: notIn
+              left:
+                value:
+                  simple: inputs.IP
+                iscontext: true
+              right:
+                value:
+                  simple: PrismaSase.AddressGroup.addresses
+                iscontext: true
+          transformers:
+          - operator: uniq
+      TSGID:
+        complex:
+          root: inputs.TSGID
+      limit:
+        simple: "50"
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 0
+    view: |-
+      {
+        "position": {
+          "x": 550,
+          "y": 580
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "10":
+    id: "10"
+    taskid: da760a85-42a1-4a17-8d6f-cf74f20579f6
+    type: condition
+    task:
+      id: da760a85-42a1-4a17-8d6f-cf74f20579f6
+      version: -1
+      name: Commit and push configuration to folders?
+      description: Commit and push configuration to folders?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "3"
+      Auto Commit:
+      - "7"
+      Manual:
+      - "12"
+    separatecontext: false
+    conditions:
+    - label: Auto Commit
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: inputs.AutoCommit
+            iscontext: true
+          right:
+            value:
+              simple: "True"
+          ignorecase: true
+    - label: Manual
+      condition:
+      - - operator: isEqualString
+          left:
+            value:
+              simple: inputs.AutoCommit
+            iscontext: true
+          right:
+            value:
+              simple: "False"
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 550,
+          "y": 910
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "12":
+    id: "12"
+    taskid: 5161b42f-d127-441e-854b-319408faaee8
+    type: condition
+    task:
+      id: 5161b42f-d127-441e-854b-319408faaee8
+      version: -1
+      name: Complete this task to push the configuration
+      description: Complete this task to push the configuration.
```

Comment:
It's not a manual flow but semi-automatic.
We need to change the name of the playbook flow (from manual to semi-automated). And add another flow for fully manual flow.
I guess its true for more playbooks

Reply:
Or - you can change a bit the question and description that if they chose the "push" it will done automatically (and immediately)

Reply:
Changed the task name to -  `Choose to commit and push configuration` on all the PBs per your request.

Reply:
Done